### PR TITLE
fix(FEC-13490): Move manager services from common into ui-managers

### PR DESCRIPTION
**Pull request by SivanA-Kaltura**
### Description of the Changes

Move managers from common to ui-managers

Related PRs:
https://github.com/kaltura/playkit-js-common/pull/40
https://github.com/kaltura/playkit-js-qna/pull/331
https://github.com/kaltura/playkit-js-hotspots/pull/310
https://github.com/kaltura/playkit-js-moderation/pull/65
https://github.com/kaltura/playkit-js-downloads/pull/34
https://github.com/kaltura/playkit-js-ivq/pull/110
https://github.com/kaltura/playkit-js-call-to-action/pull/3

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/services/banner-manager/ui/banner-container/banner-container.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      <div className={styles.bannerContainerRoot} aria-live="polite">
+        <div
+          style={`
+                background-color:${backgroundColor}; 
+                backdrop-filter: blur(${blur});
+             `}
+          className={styles.bannerContainer}
+        >
+          <button className={styles.closeButton} onClick={props.onClose}>
+            <CloseSmall className={styles.small} />
```
Comment (semarche-kaltura) [inline src/services/banner-manager/ui/banner-container/banner-container.tsx:30]: can we use close icons from common repo?

File: src/services/floating-manager/floating-manager.tsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+      this._updateCachedCanvas();
+      this._updateComponents();
+    });
+  }
+
+  public injectFloatingManager() {
+    //
+  }
+
+  // public registerUIComponents() {
```
Comment (semarche-kaltura) [inline src/services/floating-manager/floating-manager.tsx:215]: redundant code

File: src/services/preset-manager/preset-manager.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+export interface VideoResizeEvent {
+  type: PresetManagerEventTypes.VideoResizeEvent;
+}
+
+export type PresetManagerEvents = PresetResizeEvent | VideoResizeEvent;
+
+export class PresetManager {
+  private _events: EventsManager<PresetManagerEvents> = new EventsManager<PresetManagerEvents>();
+  private _items: PresetItem[] = [];
+  private _pendingItems: PresetItem[] = [];
```
Comment (semarche-kaltura) [inline src/services/preset-manager/preset-manager.tsx:34]: _pendingItems not uses in class

File: src/services/preset-manager/preset-manager.tsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+      this._kalturaPlayer.ui.addComponent({
+        label,
+        presets,
+        container,
+        get
+      });
+    }
+  }
+
+  // // TODO check if this works
```
Comment (semarche-kaltura) [inline src/services/preset-manager/preset-manager.tsx:98]: commented code

File: src/services/preset-manager/preset-manager.tsx [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  public on: EventsManager<PresetManagerEvents>['on'] = this._events.on.bind(this._events);
+  public off: EventsManager<PresetManagerEvents>['off'] = this._events.off.bind(this._events);
+
+  public add(data: PresetItemData): void {
+    const component = new PresetItem({
+      kalturaPlayer: this._kalturaPlayer,
+      data
+    });
+
+    // this._pendingItems.push(component);
```
Comment (semarche-kaltura) [inline src/services/preset-manager/preset-manager.tsx:82]: commented code

File: src/services/toast-manager/ui/toast/toast.tsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    }
+  }
+
+  render() {
+    const { text, title, icon } = this.props;
+
+    return (
+      <div className={styles.toastWrapper + ' ' + this._getToastSeverityClass()} onClick={this._onClick}>
+        <button className={styles.closeButton} onClick={this._onClose}>
+          <CloseIcon />
```
Comment (semarche-kaltura) [inline src/services/toast-manager/ui/toast/toast.tsx:59]: can we use close icon from common?

File: src/services/toast-manager/ui/toasts-container/index.ts
Comment (semarche-kaltura) [inline src/services/toast-manager/ui/toasts-container/index.ts:1]: empty file

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]

File: src/services/toast-manager/ui/toast/toast.tsx (hunk 6 above)
Comment (SivanA-Kaltura) [reply src/services/toast-manager/ui/toast/toast.tsx:59]: yeah, i just copied the existing code

**Pull request merged**